locations/spiders/wyndham.py

```diff
@@ -0,0 +1,187 @@
+from locations.items import GeojsonPointItem
+
+BRAND_MAP = {
+    "hj": "hojo",
+    "HJ": "hojo",
+    "hojo": "HJ",
+    "lq": "laquinta",
+    "LQ": "laquinta",
+    "laquinta": "LQ",
+    "di": "days-inn",
+    "DI": "days-inn",
+    "days-inn": "DI",
+    "bh": "hawthorn-extended-stay",
+    "BH": "hawthorn-extended-stay",
+    "hawthorn-extended-stay": "BH",
+    "hr": "wyndham",
+    "HR": "wyndham",
+    "wyndham": "HR",
+    "wg": "wingate",
+    "WG": "wingate",
+    "wingate": "WG",
+    "se": "super-8",
+    "SE": "super-8",
+    "super-8": "SE",
+    "bu": "baymont",
+    "BU": "baymont",
+    "baymont": "BU",
+    "dx": "dolce",
+    "DX": "dolce",
+    "dolce": "DX",
+    "dz": "dazzler",
+    "DZ": "dazzler",
+    "dazzler": "DZ",
+    "wr": "wyndham-rewards",
+    "WR": "wyndham-rewards",
+    "wyndham-rewards": "WR",
+    "kg": "knights-inn",
+    "KG": "knights-inn",
+    "knights-inn": "KG",
+    "wt": "tryp",
+    "WT": "tryp",
+    "tryp": "WT",
+    "aa": "americinn",
+    "AA": "americinn",
+    "americinn": "AA",
+    "all": "wyndham-hotel-group",
+    "ALL": "wyndham-hotel-group",
+    "wyndham-hotel-group": "ALL",
+    "ce": "caesars-entertainment",
+    "CE": "caesars-entertainment",
+    "caesars-entertainment": "CE",
+    "mt": "microtel",
+    "MT": "microtel",
+    "microtel": "MT",
+    "gn": "wyndham-garden",
+    "GN": "wyndham-garden",
+    "wyndham-garden": "GN",
+    "gr": "wyndham-grand",
+    "GR": "wyndham-grand",
+    "wyndham-grand": "GR",
+    "es": "esplendor",
+    "ES": "esplendor",
+    "esplendor": "ES",
+    "ra": "ramada",
+    "RA": "ramada",
+    "ramada": "RA",
+    "re": "registry-collection",
+    "RE": "registry-collection",
+    "registry-collection": "RE",
+    "tl": "travelodge",
+    "TL": "travelodge",
+    "travelodge": "TL",
+    "vo": "wyndham-vacations",
+    "VO": "wyndham-vacations",
+    "wyndham-vacations": "VO",
+    "tq": "trademark",
+    "TQ": "trademark",
+    "trademark": "TQ",
+}
+BRAND_TIER_MAP = {"hr": "wy", "gr": "wy", "gn": "wy", "dz": "fe", "es": "fe"}
+COUNTRIES = {
+    "Canada": "CA",
+    "Turkey": "TR",
+    "United States": "US",
+    "Mexico": "MX",
+    "Honduras": "HN",
+    "Chile": "CL",
+    "Colombia": "CO",
+}
+HEADERS = {
+    "Host": "www.wyndhamhotels.com",
+    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:81.0) Gecko/20100101 Firefox/81.0",
+    "Accept": "*/*",
+    "Accept-Language": "en-US,en;q=0.5",
+    "Accept-Encoding": "gzip, deflate, br",
+    "X-Requested-With": "XMLHttpRequest",
+    "DNT": "1",
+    "Connection": "keep-alive",
+    "Referer": "https://www.wyndhamhotels.com/locations",
+    "TE": "Trailers",
+}
+
+
+
+def create_url(brand, city, state, unique_url, tier_id):
+    # In the html, there is a script which this recreates
+    brand_name = BRAND_MAP.get(brand)
+    if not brand_name:
+        brand_name = BRAND_MAP.get(tier_id.lower())
+    url = brand_name + "/"
+    url += city.replace(" ", "-").lower()
```

Comment:
It looks like some of the cities have a `.` in the name that needs to be removed, too. Check the logs here for examples: https://placescraper-results.s3.amazonaws.com/ci/319483762/wyndham/log.txt

Reply:
log is handy; will check those on failures, thanks.. updated to remove "`."`